### PR TITLE
Prepare release - 1.0.0

### PR DESCRIPTION
**Pull request by VictorAlbertos**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## 0.1.3
```
Comment (chemarubio) [inline CHANGELOG.md:1]:
```suggestion
## 1.0.0
```

File: android/build.gradle [hunk 2]
```diff
@@ -61,13 +61,13 @@ android {
...
         minSdkVersion 24
-        targetSdkVersion 33
+        targetSdkVersion 34
     }
 }
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation("com.significo:recco-api-ui:0.0.6")
+    implementation("com.significo:recco-api-ui:1.0.0")
```
Comment (chemarubio) [inline android/build.gradle:70]: I guess recco.podspec should be in line with the 1.0.0 version

**Review by chemarubio — APPROVED** [review]
🚀

File: pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: recco
 description: Recco is a personalized, and user-friendly approach to health and wellness recommendations based on users' unique interests, motivations, and health behaviors.
-version: 0.1.3
+version: 1.0.0
```
Comment (chemarubio) [inline pubspec.yaml:3]: I guess recco.podspec should be in line with the 1.0.0 version

Comment (VictorAlbertos) [reply pubspec.yaml:3]: @arubio will do that change

**Pull request merged**